packages/features/webhooks/components/WebhookForm.tsx

```diff
@@ -32,6 +32,8 @@ const WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP_V2: Record<string, WebhookTriggerEve
   core: [
     { value: WebhookTriggerEvents.BOOKING_CANCELLED, label: "booking_cancelled" },
     { value: WebhookTriggerEvents.BOOKING_CREATED, label: "booking_created" },
+    { value: WebhookTriggerEvents.BOOKING_REJECTED, label: "booking_rejected"},
+    { value: WebhookTriggerEvents.BOOKING_REQUESTED, label: "booking_requested"},
```

Comment:
Heads up when creating new labels don't forget to include the string in `apps/web/public/static/locales/en/common.json`

<img width="796" alt="image" src="https://github.com/calcom/cal.com/assets/65426560/f1d6998c-7c6c-493b-a5f2-a98f00fddb4e">

Reply:
Thanks for the heads up :D